recipes-bsp/jh7110-spl-tool/jh7110-spl-tool_git.bb

```diff
@@ -0,0 +1,25 @@
+DESCRIPTION = "spl_tool is a jh7110 signature tool used to generate spl header information and generate u-boot-spl.bin.normal.out."
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=e6dc25dc2418b8831c906d43809d8336"
+SECTION = "BSP"
```

Comment:
Make this to match u-boot
`SECTION = "bootloaders"`

Reply:
ok